js/test/date/testdatefmt_be_BY.js

```diff
@@ -0,0 +1,1788 @@
+        test.expect(2);
+        var fmt = new DateFmt({locale: "be-BY", type: "time", length: "full", time: "h"});
+        test.ok(fmt !== null);
+        
+        var date = new GregorianDate({
+            locale: "be-BY",
+            year: 2011,
+            month: 9,
+            day: 29,
+            hour: 13,
+            minute: 45,
+            second: 37,
+            millisecond: 0
+        });
+        test.equal(fmt.format(date), "13");
+        test.done();
+    },
+
+    testDateFmtFullTimeComponentsMS_be_BY: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({locale: "be-BY", type: "time", length: "full", time: "ms"});
+        test.ok(fmt !== null);
+        
+        var date = new GregorianDate({
+            locale: "be-BY",
+            year: 2011,
+            month: 9,
+            day: 29,
+            hour: 13,
+            minute: 45,
+            second: 37,
+            millisecond: 0
+        });
+        test.equal(fmt.format(date), "45:37");
+        test.done();
+    },
+
+    testDateFmtFullTimeComponentsHM_be_BY: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({locale: "be-BY", type: "time", length: "full", time: "hm"});
+        test.ok(fmt !== null);
+        
+        var date = new GregorianDate({
+            locale: "be-BY",
+            year: 2011,
+            month: 9,
+            day: 29,
+            hour: 13,
+            minute: 45,
+            second: 37,
+            millisecond: 0
+        });
+        test.equal(fmt.format(date), "13:45");
+        test.done();
+    },
+
+    testDateFmtFullTimeComponentsHMS_be_BY: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({locale: "be-BY", type: "time", length: "full", time: "hms"});
+        test.ok(fmt !== null);
+        
+        var date = new GregorianDate({
+            locale: "be-BY",
+            year: 2011,
+            month: 9,
+            day: 29,
+            hour: 13,
+            minute: 45,
+            second: 37,
+            millisecond: 0
+        });
+        test.equal(fmt.format(date), "13:45:37");
+        test.done();
+    },
+
+    testDateFmtFullTimeComponentsHMA_be_BY: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({locale: "be-BY", type: "time", length: "full", time: "hma"});
+        test.ok(fmt !== null);
+        
+        var date = new GregorianDate({
+            locale: "be-BY",
+            year: 2011,
+            month: 9,
+            day: 29,
+            hour: 13,
+            minute: 45,
+            second: 37,
+            millisecond: 0
+        });
+        test.equal(fmt.format(date), "13:45");
+        test.done();
+    },
+
+    testDateFmtFullTimeComponentsHMZ_be_BY: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({
+            locale: "be-BY", 
+            type: "time", 
+            length: "full", 
+            time: "hmz", 
+            timezone: "Europe/Moscow"
```

Comment:
The time zone used in Belarus is "Europe/Minsk", which is similar to "Europe/Moscow" but different.